src/Lecture2.hs

```diff
@@ -97,7 +121,8 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+dropSpaces :: [Char] -> [Char]
+dropSpaces l = takeWhile (not . isSpace) (dropWhile isSpace l)
```

Comment:
Looks amazing! 🏆 

You can eta-reduce this function by using one more function c⚪mp⚪sition

  ```suggestion
dropSpaces = takeWhile (not . isSpace) . dropWhile isSpace
```